admin-dev/themes/new-theme/js/components/grid/extension/filters-submit-button-enabler-extension.js

```diff
@@ -37,7 +37,7 @@ export default class FiltersSubmitButtonEnablerExtension {
     const $filtersRow = grid.getContainer().find('.column-filters');
     $filtersRow.find('.grid-search-button').prop('disabled', true);
 
-    $filtersRow.find('input, select').on('input', () => {
+    $filtersRow.find('input, select, datepicker').on('input dp.change', () => {
```

Comment:
No breaking change possible for the pages that already use this extension ?

Reply:
Noticed that the selector of datepicker was incorrect :smile:  and in fact its not even needed (as it's already selected as input). Anyway, this only adds additional datepicker event and it is the first migrated page to have datepicker filter in grid so i doubt it could cause bc break.